### PR TITLE
Topic2

### PR DESCRIPTION
**Pull request by sanchezignacio025**
I have sent the exercise topic 2

### CODE REVIEW

File: ejercicio2/src/test/java/practica2/AddPost.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+public class AddPost {
+
+    public Post[] addNewPost(){
+		
+		Post[] post = PostTest.getPost();
+		String newNote;
+		
+		int count =0; 
+		for (int i = 0; i <= 9; i++){
```
Comment (gastonabastoG) [inline ejercicio2/src/test/java/practica2/AddPost.java:13]: The for clause is empty

File: ejercicio2/src/test/java/practica2/DeletedFile.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import java.util.Scanner;
+
+public class DeletedFile {
+	public Post[] deletedFile(){
+		
+		Post[] post = PostTest.getPost();
+		int deletedFileNumber=-1;
+		System.out.println("delete note: ");
+		Scanner scan = new Scanner(System.in);
+		deletedFileNumber =scan.nextInt();
```
Comment (gastonabastoG) [inline ejercicio2/src/test/java/practica2/DeletedFile.java:12]: What do you do with this variable?

File: ejercicio2/src/test/java/practica2/PostTest.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+		post[1] = new Post("note 2");
+		post[2] = new Post("note 3");
+		post[3] = new Post("note 4");
+		post[4] = new Post("note 5");
+		post[5] = new Post("note 6");
+		post[6] = new Post("note 7");
+		post[7] = new Post("note 8");
+		post[8] = new Post("note 9");
+		post[9] = new Post("note 10");
+		for (int i = 0; i <= 9; i++){
```
Comment (gastonabastoG) [inline ejercicio2/src/test/java/practica2/PostTest.java:25]: Why do you create Post with null value?

File: ejercicio2b/src/test/java/TD2/RecentFileTest.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+		 for (int i = 0; i <= 14; i++)   {
+			System.out.println("number : " + num[i]);
+			index++;
+		}
+		
+		
+	}
+	
+
+	public static int[] getNum() {
```
Comment (gastonabastoG) [inline ejercicio2b/src/test/java/TD2/RecentFileTest.java:59]: Why did you use static methods?

File: ejercicio2b/src/test/java/TD2/RecentFileTest.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+	static int[] num = null;
+	
+	@Test
+    public void showEmptyList1() {
+
+		
+		int[] num = null;
+		if (num == null) {
+			System.out.println("The list of files is empty");
+		}
```
Comment (gastonabastoG) [inline ejercicio2b/src/test/java/TD2/RecentFileTest.java:18]: You should use assert sentence

**Review by gastonabastoG — COMMENTED** [review]
Revise my comments

File: ejercicio2/src/test/java/practica2/PostTest.java (hunk 3 above)
Comment (sanchezignacio025) [reply ejercicio2/src/test/java/practica2/PostTest.java:25]: Because the topic say that: "When the program is run for the first time, the list is empty"

**Pull request merged**